frontend/app/models/branch-adress.js

```diff
@@ -0,0 +1,7 @@
+import DS from "ember-data";
+
+export default DS.Model.extend({
+  short_name: DS.attr("string"),
+  adress_information: DS.attr("string"),
```

Comment:
use `-` instead of `_`

Reply:
With `-` we get a build error so I doubt that that is correct.
I've changed it to camelCase for now as it seems more inline with the rest of the tool.